openblas-build/src/build.rs

```diff
@@ -394,9 +394,11 @@ impl Configure {
         }
         if self.use_thread {
             args.push("USE_THREAD=1".into());
+            args.push("NUM_THREADS=256".into());
```

Comment:
I’m not certain if it’s necessary . After setting use_thread, will OpenBlas makefile script detect num_thread atomically?